src/cli/cli.hpp

```diff
@@ -494,14 +524,14 @@ class Interpreter
                                             otMessage *          aMessage,
                                             const otMessageInfo *aMessageInfo,
                                             void *               aContext);
-    void        OutputSpaces(uint16_t aCount);
-    void        OutputMode(const otLinkModeConfig &aMode, uint16_t aColumn);
-    void        OutputConnectivity(const otNetworkDiagConnectivity &aConnectivity, uint16_t aColumn);
-    void        OutputRoute(const otNetworkDiagRoute &aRoute, uint16_t aColumn);
-    void        OutputRouteData(const otNetworkDiagRouteData &aRouteData, uint16_t aColumn);
-    void        OutputLeaderData(const otLeaderData &aLeaderData, uint16_t aColumn);
-    void        OutputNetworkDiagMacCounters(const otNetworkDiagMacCounters &aMacCounters, uint16_t aColumn);
-    void        OutputChildTableEntry(const otNetworkDiagChildEntry &aChildEntry, uint16_t aColumn);
+
+    void OutputMode(uint16_t aIndentSize, const otLinkModeConfig &aMode);
```

Comment:
```suggestion
    void OutputMode(uint8_t aIndentSize, const otLinkModeConfig &aMode);
```
nit: unit8_t is enough?

Reply:
changed to` uint8_t